### ISSUE TITLE
Is Hindi Translation really needed?

### ISSUE THREAD

**Issue by hamees-sayed** [brigadecore/docs-hindi#4]
Indians don't necessarily speak or read such an authentic Hindi, even more, when we type Hindi it's usually in English since Hindi is phonetic and can easily be written in English as the "sounds" of the words. 

I really don't think Hindi translation will add any major value to docs, instead, the effort going into that translation should be better spent elsewhere is what I feel.

**Comment by krancour** [comment]
Note that the org maintainers aren't really fit to make a decision on this.

@a-muk and @shubham1172 are responsible for this repo and I'll let them field this question.

**Comment by shubham1172** [comment]
@krancour the need for this repository must have been suggested somewhere, right? Do we have any data on how many people are asking for it? I agree with @hamees-sayed that a lot of Indians do not speak Hindi. Another aspect to it - the technologies that brigade is built around do not have Hindi docs in the first place. This means that brigade users might be already familiar with English. However, this also gives us the opportunity to take the first step towards introducing Hindi docs in this ecosystem.

I would propose asking the community and only continuing with this if we get a positive response.

**Comment by hamees-sayed** [comment]
@shubham1172 Agreed, adding Hindi in docs for more inclusiveness is a good idea but we certainly need the community to decide for themselves if efforts put into Hindi translation is actually worth it or not...

**Comment by krancour** [comment]
> the need for this repository must have been suggested somewhere, right?

@a-muk volunteered so it became a thing.

> Do we have any data on how many people are asking for it?

tbh, not many people are using Brigade atm. We're _just_ starting to reach a tipping point where there is increased interest in the project.

**Comment by attreyee-muk** [comment]
I think that writing the docs in languages people are comfortable with can help including them more in the community. Not many people in India have absolute command over English . So, translating the docs in various languages can prove beneficial.

**Comment by shubham1172** [comment]
If there is less community interest, we might have a hard time maintaining this repo in the long run too.

**Comment by attreyee-muk** [comment]
Yeah, true. But nothing can be said right now on that, can it be?

**Comment by krancour** [comment]
fwiw, I am unopposed to voluntary translations to any language if _someone_ finds it worthwhile. I do think it helps with building an inclusive community.

I do have to work out how we notify the maintainers of each translation when the English docs are updated, so they can make the corresponding update.

It's a bridge we can cross if/when we come to that, but it's possible we could retract translations or add a disclaimer to them if they become too stale over time.

Bottom line is I don't know that we _need_ this translation, but I doubt that it's hurting us to do it.

**Comment by shubham1172** [comment]
Agreed. 

> I do have to work out how we notify the maintainers of each translation when the English docs are updated, so they can make the corresponding update.

We can add a GH action to create issue in all the docs repositories whenever an issue is created to update English docs. Now there can be updates which don't require other docs repos to update (for example, typo fixes), so the GH action can look for certain keywords/tags in the main issue.

**Comment by krancour** [comment]
> We can add a GH action...

Or we could use Brigade since Brigade's an automation platform and we already automate everything else using Brigade. 😄

**Comment by krancour** [comment]
Ok... to put closure on this, I think we mostly agree this isn't strictly needed, but it's still welcome.

At minimum, I am happy for everything @a-muk has learned about Brigade in the process, because I think she may be eager to make other contributions as well.

No rush, but #2 can proceed and I will update and merge https://github.com/brigadecore/brigade/pull/1880 whenever it's ready.

**Comment by attreyee-muk** [comment]
Thank you @krancour and @shubham1172 . I really learned a lot about Brigade while doing the translation. I look forward to contributing to other issues.
